libs/sdk-ui-tests-e2e/cypress/tools/widget.ts

```diff
@@ -77,6 +77,11 @@ export class Widget {
         return this;
     }
 
+    confirmDelete() {
```

Comment:
write this function inside removeKPIWidget, add 1 param confirm has default value = true
if confirm then click delete
else click cancel 

removeKPIWidget(confirm=true)